neo/Ledger/MemoryPool.cs

```diff
@@ -253,6 +255,34 @@ public IEnumerable<Transaction> GetSortedVerifiedTransactions()
             }
         }
 
+        // This function return some statistics about the best _maxTxPerBlock and _maxLowPriorityTxPerBlock txs of the mempool
+        public JObject GetMemPoolInfoForNextBlock()
```

Comment:
Better name would be `GetStatisticsForNextBlock`

Reply:
Or actually, since I'm suggesting to not make it specific to anything about the next block, just `GetStatistics`

Reply:
I agree, Jeff.

GetStatisticsForNextBlock sounds good.
I think that get info for the whole mempool does not sound so much useful.